module.prop

```diff
@@ -1,6 +1,6 @@
 id=emoji-joypixels
 name=JoyPixels®
-version=v6.1
-versionCode=2
-author=shivangswain
-description=Systemlessly replaces emoji font with JoyPixels 6.1.
+version=v6.5
+versionCode=3
+author=shivangswain, RainySummer
```

Comment:
I request you to remove your name from the author tag in the module.prop file. I'd rather add your name to the top of the README under a contributors section.